Prepare document for AFP submission

benkeks opened the pull request.

benkeks committed:
Update affiliations and begin document cleanup

benkeks committed:
Remove file coverage check from workflow

benkeks committed:
Increase workflow timeout

benkeks committed:
Remove intro and conclusion

benkeks committed:
Fix ROOT file

benkeks committed:
Provide some overview

benkeks committed:
Start to adapt the spectroscopy game to use Caro's energy game theory

benkeks committed:
Revert "Start to adapt the spectroscopy game to use Caro's energy game theory"

This reverts commit 6776337a9f71964bb454135db772ecbd832952d8.

It seems to much of a change to use the existing Galois energy games due to changed defs.
Also, it requires at least one change to the Galois energy game AFP entry to go through.

benkeks committed:
Drop example instantiation

benkeks committed:
Drop manual page breaks

benkeks committed:
Make spectroscopy game into a sublocale

benkeks committed:
Make lts locale naming more consistent

benkeks committed:
Use nicer syntax highlighting in LaTeX

benkeks committed:
Simplify weak trace theory

benkeks committed:
Simplify presentation of formulas and expressiveness prices

benkeks committed:
Improve PDF linebreaks of Sec 3

benkeks committed:
Make weak trace theory a little nicer

benkeks committed:
Improve formatting of eta and branching bisim theories

benkeks committed:
Fix line breaks and comments in later theory parts

benkeks committed:
Provide some overview explanation of the theory

benkeks committed:
Mention equiv.io

crmrtz committed:
minimal changes to the presentation

(removing some unnecessary lemmas)

crmrtz commented:
Hey Ben,

Awesome that you revisited the Isabelle theories and are polishing them up for the AFP. Looks good to me! 

Hope the Palaiseau sunshine makes a rare appearance for you :)

crmrtz committed:
comply with afp submission guidelines

"Timeouts may be no finer than five-minute increments, i.e. must be divisible by 300."

ekeln committed:
small spelling mistake that bugged me

benkeks committed:
Get rid of unused Misc theory

benkeks committed:
Become a little more formal in tone

benkeks committed:
Fix some more PDF line breaks

benkeks committed:
Remove trailing whitespaces

benkeks committed:
Add pointer for gentle introduction

benkeks committed:
Add license headers to files as required by the AFP; update affiliations
